tests/dropna_test.py

```diff
@@ -112,16 +112,26 @@ def test_dropna():
     assert (df.s.dropna().tolist() == ["aap", "noot", "mies"])
     assert (df.o.dropna().tolist() == ["aap", "noot"])
 
-
-def test_dropna_all_columns():
-    x = [1, 2, 3, 4, 5]
-    y = ['dog', 'dog', None, 'cat', None]
+@pytest.fixture
+def df_with_missings():
+    x = [1.1, np.nan, np.nan, 4.4, 5.5]
+    y = ["dog", "dog", None, "cat", None]
```

Comment:
OK re: quotes :) PS have you considered applying black formatting? I love it. It can solve some merge conflicts as a in addition to (in my opinion) making it more readable.

Does this factory exist already? If so, seems like it. Even if not, this seems like a worthwhile investment to write it, seems like it would/should get used all over the place.

Reply:
Regarding black: yes we've considered it, but we will probably not go for it. We (or at least I) do not agree with all its stuff, and find it too restrictive. I do use it for some other projects tho. But for vaex, nah. 
The quotes thing is just a personal preference tho :P (I find the single ones more aesthetically pleasing, and since I look at this a lot.. :D ).

As for the factory stuff, please look at common.py in the test folder. We also used it in the new tests for `where`, which are in compute_test.py  (see e.g. `test_where_primitive_masked_condition`)

Reply:
Cool, re black, no problem!

I'll take a look at the factory